### ISSUE TITLE
Amuse package requirements

### ISSUE THREAD

**Issue by rieder** [amusecode/amuse#412]
Currently, amuse-framework has all the AMUSE prerequisites, though it only needs Python (and python related things) itself. 
Maybe we can be more flexible about this in the future - e.g. amuse-framework has very few requirements, C++-based codes require a C++-compiler, Fortran codes a Fortran compiler, etc.
This would probably require splitting up the configure script (or at least its output).

**Comment by ipelupessy** [comment]
I find this tricky, since you can also use the installed amuse framework as a basis to develop....
(though some libraries might be dropped)

**Comment by rieder** [comment]
How about splitting amuse-framework one more time, into
- amuse-core (only the Python bits that need no compiling or configuring at all)
- amuse-libraries (the other bits)

**Comment by ipelupessy** [comment]
yes, I like the idea. I would retain amuse-framework as high level package with these two. Maybe a better name for amuse-libraries??  (don't have a picture of the exact split..)

**Comment by rieder** [comment]
I'll have a go at this.

**Comment by ipelupessy** [comment]
ok, be careful though..I think the config is required in unexpected places...
